plugins/chat/assets/javascripts/discourse/components/chat-live-pane.hbs

```diff
@@ -40,17 +40,16 @@
 
   <div
     class="chat-messages-scroll chat-messages-container"
-    {{chat/on-throttled-scroll this.computeScrollState (hash delay=50)}}
+    {{on "scroll" this.computeScrollState passive=true}}
     {{chat/on-throttled-scroll this.resetIdle (hash delay=500)}}
+    {{chat/on-throttled-scroll this.computeArrow (hash delay=150)}}
```

Comment:
I prefer to have more scroll handlers but divide them by criticity, the un-throttled one give us exact precision but has to be kept ultra fast.